molecularnodes/ui/ops.py

```diff
@@ -734,6 +734,11 @@ class MN_OT_Import_Trajectory(bpy.types.Operator):
         description="Add nodes to the scene to load the trajectory",
         default=True,
     )
+    use_dssp: BoolProperty(  # type: ignore
+        name="Use DSSP",
+        description="Assign secondary structure using the DSSP algorithm",
+        default=False,
```

Comment:
I think we can probably have the `default=True` as most users would want secondary structure for their visualisations.